share/eds/pubsub.go

```diff
@@ -57,23 +57,10 @@ func (s *PubSub) Start(context.Context) error {
 // * Unregisters all the added Validators
 // * Closes the `ShrEx/Sub` topic
 func (s *PubSub) Stop(context.Context) error {
-	err := s.pubSub.UnregisterTopicValidator(s.pubSubTopic)
-	if err != nil {
-		return err
-	}
-
+	// TODO(@vgonkivs): unregister the topic validator
```

Comment:
Why does it get removed here?

Reply:
We've not had a subscriber yet. So, no one is registering a validator rn. Call `UnregisterValidator` fails if there is no validator.

Register/Unregister validator should be added in a PR with the subscribing to the "eds-sub".

Reply:
👍🏻